grammar/fbp.peg

```diff
@@ -17,6 +17,93 @@
 
   nodes = {};
 
+  var serialize, indexOf = [].indexOf || function(item) { for (var i = 0, l = this.length; i < l; i++) { if (i in this && this[i] === item) return i; } return -1; };
+
+  parser.serialize = function(graph) {
```

Comment:
How I think this should be solved is to let `lib/fbp.js ` be a concatenation of
* the JS generated by PEG
* the JS generated from CS
* one liner which sets `module.exports.serialize`

The intermediate .ja files should not be checked into git, only the last result.